internal/scheduler/query.sql.go

```diff
@@ -0,0 +1,868 @@
+// Code generated by sqlc. DO NOT EDIT.
+// versions:
+//   sqlc v1.13.0
+// source: query.sql
+
+package scheduler
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+)
+
+const getTopicMessageIds = `-- name: GetTopicMessageIds :many
+SELECT topic, ledger_id, entry_id, batch_idx, partition_idx FROM pulsar WHERE topic = $1
+`
+
+func (q *Queries) GetTopicMessageIds(ctx context.Context, topic string) ([]Pulsar, error) {
```

Comment:
let's delete this (and other pulsar-db stuff)

Reply:
Agreed. Deleted.